homeassistant/components/unifi/strings.json

```
@@ -46,6 +46,15 @@
         "description": "Configure client controls\n\nCreate switches for serial numbers you want to control network access for.",
         "title": "UniFi options 2/3"
       },
+      "simple_options": {
+        "data": {
+          "track_clients": "Track network clients",
+          "track_devices": "Track network devices (Ubiquiti devices)",
+          "block_client": "Network access controlled clients"
```

Please use references.
```suggestion
          "track_clients": "[%key:components::unifi::options::step::device_tracker::data::track_clients%]",
          "track_devices": "[%key:components::unifi::options::step::device_tracker::data::track_devices%]",
          "block_client": "[%key:components::unifi::options::step::device_tracker::data::block_client%]"
```

---

Didn't know it was ready 🎉

---

It has actually always worked, we just never did 🙈